src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2004,7 +2005,34 @@ bool entitiesQuery
   auto_ptr<DBClientCursor>  cursor;
   Query                     query(finalQuery.obj());
 
-  query.sort(BSON(ENT_CREATION_DATE << 1));
+  if (sorted == "")
```

Comment:
Yeah, I understood all this ... after a while (a long one :-)).

The name is very confusing, that's all.
Had it been called sortOrderList, then it would have been easier